saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -27,6 +27,16 @@ wait_for_authorized_keys() {
 }
 
 setup_tmp_ssh() {
+
+  if ! id ${SSH_USER}; then
```

Comment:
I'm not sure why is this needed here as we have similar functionality in the prerequisites:
https://github.com/hortonworks/cloudbreak-images/blob/master/saltstack/base/salt/prerequisites/user.sls

Reply:
This patch is based on rc-1.16 where those modifications do not exist. Removing it.